### PR TITLE
79 star component

### PR DESCRIPTION
**Pull request by mindfulme**
#### What's this PR do?
The PR renames the branch from 'star-component' to '79-star-component' also adding useful corrections to Star.vue, reference to [the old PR](https://github.com/bitshares/bitshares-community-ui/pull/94) 
#### Where should the reviewer start?
Star.vue
#### How should this be manually tested?
The star should be filled on hover
http://localhost:8080/#/components (if locally)
or
http://staging-ui.trusty.apasia.tech:8080/79-star-component/#/components 

#### Any background context you want to provide?
The star hovers on select, also can be permanently selected from props
#### What are the relevant tickets?
[#79](https://github.com/bitshares/bitshares-community-ui/issues/79)
#### Screenshots (if appropriate)
screen shot 2018-10-21 at 09 10 54

#### URL
Provide a link to your deploy from http://staging-ui.trusty.apasia.tech:8080/79-star-component/#/components
#### Questions:
Hover should be done from CSS or by replacing an icon using js? I had a solution using js.

### CODE REVIEW

**Comment by roma219** [comment]
@MindfulMe Why did you close preview PR? Now the hover is done ok, but the hover and active state's color are the same...

**Comment by mindfulme** [comment]
@roma219 The old PR was closed due to the bad branch name (you noted that in the past, to give branches more appropriate names), also I will make the white color from props and grey from hover while not filled from props, is it ok?

File: src/components/Star/Star.vue [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+}
+</script>
+
+<style lang="scss" scoped>
+.star {
+  cursor: pointer;
+  color: config('colors.star');
+}
+
+.defaultActive {
```
Comment (roma219) [inline src/components/Star/Star.vue]: Name classes in the same codestyle, and use better naming that is self explaniatory. It's hard to understand what is  "default active" (default what?). Better name it `is-active`

**Review by roma219 — CHANGES_REQUESTED** [review]
Last change and it's ok

**Comment by roma219** [comment]
@MindfulMe 
image

**Review by roma219 — APPROVED** [review]

**Pull request merged**